oneview_redfish_toolkit/api/zone.py

```diff
@@ -45,7 +45,15 @@ def __init__(self, profile_template, available_targets_obj):
         """
         super().__init__(self.SCHEMA_NAME)
 
-        available_targets = available_targets_obj["targets"]
+        self.available_targets = available_targets_obj["targets"]
```

Comment:
Is there a reason to this be an instance attribute and not just a variable? I see it is used inside private methods, but it could be passed as argument, not creating code dependency. Now, methods like "fill_resource_blocks()" depends that instance variable "self.available_targets" has be set before call the method.

Reply:
No special reason, I has used instance attributes because the method that are using them should be a non public method, so should be called only inside the class and is being called by the constructor that know how prepare the instance variable correctly.
But, I can do the requested changes. =)